### DISCUSSION TITLE
Typescript

### DISCUSSION THREAD

**Discussion by faraplay** [OctoNezd/oldlander#48] [Ideas]
I notice that a Typescript file was added in commit 259c230b8e0f0301d4715733ee62ab7d13662d7c.
We could convert the current code to Typescript if we're going to be using it. Honestly I wouldn't mind going through all the code myself and adding types to everything.

**Comment by OctoNezd** [comment]
This would be great - I am not particularly good at TypeScript

**Reply by faraplay** [reply]
Alright, I'll wait for you to do the file restructuring first before starting to avoid lots of merge conflicts.
